src/main/scala/com/kinja/amqp/model/Message.scala

```diff
@@ -2,14 +2,26 @@ package com.kinja.amqp.model
 
 import java.sql.Timestamp
 
+sealed trait MessageLike {
+	def routingKey: String
+	def exchangeName: String
+	def message: String
+	def createdTime: Timestamp
+}
+
 final case class Message(
+	routingKey: String,
```

Comment:
Using overrides can make explicit what are the parameters from the base trait.

Reply:
OK